css/style.css

```diff
@@ -1,9 +1,203 @@
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  display: flex;
+  flex-direction: column;
+  padding-left: 5%;
+  padding-right: 5%;
+  margin-left: 5%;
+  margin-right: 5%;
+}
+
+header {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: row;
+  margin-bottom: 0%;
+  padding-bottom: 0%;
+}
+
+.icodiv {
+  width: 20%;
+  margin-left: 1%;
+  padding-left: 1%;
+  margin-bottom: 0%;
+}
+
+.icoimage {
+  width: 15%;
+  height: 15%;
+}
+
+.navdiv {
+  width: 50%;
+  margin-bottom: 0%;
+}
+
+.unorderlist {
+  display: flex;
+  flex-direction: row;
+  padding: 5%;
+  justify-content: space-around;
+  list-style-type: none;
+  font-size: 20px;
+}
+
+a {
+  color: rgb(128, 118, 118);
+  text-decoration: none;
+}
+
+.meetkar {
+  font-weight: 600;
+}
+
+a:hover {
+  color: red;
+}
+
+main {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 0%;
+  margin-bottom: 10%;
+}
+
+.maindivimage {
+  position: relative;
+  width: 120%;
+  text-align: center;
+  color: white;
+}
+
+.mainimag {
+  max-width: 100%;
+  height: 80%;
+  padding: 5px;
+}
+
+.textimage1 {
+  position: absolute;
+  top: 40%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-size: 48px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.textimage2 {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-size: 24px;
+}
+
+.button {
+  position: absolute;
+  top: 70%;
+  background-color: rgb(248, 127, 14);
+  border: none;
+  color: white;
+  padding: 15px 32px;
+  transform: translate(-50%, -50%);
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 20px;
+  margin: 4px 2px;
+  cursor: pointer;
+}
+
+.bottomimages {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: row;
+  margin-top: 0%;
+  width: 100%;
+}
+
+.divtext {
+  margin: 2%;
+  padding: 5%;
+  text-align: center;
+  font-size: 44px;
+  color: black;
+  font-weight: 500;
+}
+
+.image1 {
+  width: 33.33%;
+}
+
+.bimage {
+  width: 30%;
+  align-items: center;
+  transform: translate(120%, 10%);
+}
+
+.underimagetextr {
+  font-size: 28px;
+  font-weight: 500;
+  text-align: center;
+  transform: translate(0%, 70%);
+}
+
+.solid {
+  border-top: 1px solid #bbb;
+}
+
+footer {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: column;
+}
+
+.footertext {
+  margin: 2%;
+  padding: 2%;
+  text-align: center;
+  font-size: 16px;
+  color: black;
+  font-weight: 500;
+}
+
+.footerimages {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  padding-left: 30%;
+  padding-right: 30%;
+  margin-left: 6%;
+  flex-direction: row;
+}
+
+.divfooter {
+  width: 80%;
+}
+
+.fimage {
+  width: 20%;
+  align-items: flex-end;
+  transform: translate(80%, 10%);
```

Comment:
Again, you don't need `transform` here, you just need to set your flexbox rules correctly. The rule `align-items: flex-end;` will have no effect here because `fimage` is not a flex container. 
The best solution would be to make `divfooter` a flex container by adding `display: flex` to it, and also adding the flex rules you need to center your images.